storybook/stories/API/cartesian/ErrorBar.stories.tsx

```diff
@@ -48,7 +48,7 @@ const StyleProps: Args = {
     defaultValue: 1.5,
   },
   direction: {
-    description: `Only used for ScatterChart with error bars in two directions. 
+    description: `Only used for ScatterChart with error bars in two directions.
```

Comment:
out of scope of this PR but would like to know the details of `direction` since its supposed to be specific to only scatter. My question would be to confirm if this is really true or not - does if affect Bar or Area or Line?